docs/admin/authorization/index.md

```diff
@@ -68,47 +68,66 @@ of the `bind` verb on `roles` and `clusterroles` resources in the `rbac.authoriz
 
 #### Checking API Access
 
-Kubernetes exposes the `subjectaccessreviews.v1.authorization.k8s.io` resource as a
-normal resource that allows external access to API authorizer decisions.  No matter which authorizer
-you choose to use, you can issue a `POST` with a `SubjectAccessReview` just like the webhook
-authorizer to the `apis/authorization.k8s.io/v1/subjectaccessreviews` endpoint and
-get back a response.  For instance:
+Kubernetes exposes resources in the `authorization.k8s.io` API group to query the API server's
+authorizer, which can be useful for debugging. For example, creating a `SelfSubjectRulesReview`,
+either using `kubectl` or directly with an HTTP `POST`, returns the set of actions the current
+user can perform for a given namespace. Note that the user must have permission to create a
+`SelfSubjectRulesReview` resource.
 
 ```bash
-kubectl create --v=8 -f -  << __EOF__
+kubectl create -o yaml -f - << EOF
```

Comment:
Not sure which I’d recommend here. `kubectl auth can-I` makes SAR checks very simple, and they are accurate regardless of the authorizer in use. SelfSubjectRulesReview is useful as well, but has inconsistent support across all authorizers

Reply:
Yeah, I think recommending `kubectl auth can-i` would be better. I've opened https://github.com/kubernetes/kubernetes/issues/52745 for SelfSubjectRulesReview improvements over 1.9, such as webhook support. Maybe we can recommend rules review after a few of those are fixed.